app/views/people/tasks.html.erb

```diff
@@ -0,0 +1,43 @@
+<h1><%= @person.name %>'s Tasks</h1>
+
+<table>
+  <tr>
+    <th>
+      Name
+    </th>
+    <th>
+      Description
+    </th>
+    <th>
+      Status
+    </th>
+    <th>
+      Completion Date
+    </th>
+  </tr>
+  <tbody>
+    <% @all_tasks.each do |task| %>
+      <% if task.completed_date.present? %>
+        <% @task_css = "completed" %>
+      <% else %>
+        <% @task_css = "incomplete" %>
+      <% end %>
+    <tr>
+      <td class="<%= @task_css %>">
```

Comment:
As a suggestion for reducing the code, you could apply the `@task_css` class to the `tr` element. I believe that the color inherits from the `tr` to the `td` so I don't think you'd even need to change the css files.